frontend/src/js/components/leftnav.js

```diff
@@ -130,11 +134,12 @@ export const LeftNav = () => {
   const { classes } = useStyles();
 
   const userCapabilities = useSelector(getUserCapabilities);
+  const tenantCapabilities = useSelector(getTenantCapabilities);
   return (
     <div className={`leftFixed leftNav ${classes.list}`}>
       <List style={{ padding: 0 }}>
         {listItems.reduce((accu, item, index) => {
-          if (!item.canAccess({ userCapabilities })) {
+          if (!item.canAccess({ userCapabilities }, { tenantCapabilities })) {
```

Comment:
```suggestion
          if (!item.canAccess({ userCapabilities, tenantCapabilities })) {
```
Passing just a single options object with named attributes... just to prevent having a long list of options/ working around the absence of named parameters in JS